db/seeds.js

```diff
@@ -41,6 +46,20 @@ const reset = async () => {
       ]
     }
   ])
+  const saltRounds = 10
+  const users = await User.create([ 
+    {
+      username: "user",
+      password: "$2b$10$uudM8Q/k8qXrsW99Puwj.evV.0maEhkh2f68/RYcCLdmmmyy7SANS"
```

Comment:
You could just include `bcrypt` in this file, and then do `bcrypt.hashSync()` or the equivalent.